src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -16,7 +18,7 @@
  */
 public class AddCommand extends Command {
 
-    public static final String COMMAND_WORD = "add";
+    public static final List<String> COMMAND_WORD = List.of(new String[]{"add", "a"});
```

Comment:
```suggestion
    public static final List<String> COMMAND_WORDS = List.of(new String[]{"add", "a"});
```

Perhaps change all occurrences of `COMMAND_WORD` to plural `COMMAND_WORDS` to reflect the new usage?

Reply:
Done